mod-spring-petstore/descriptors/DeploymentDescriptor-template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "srvcId": "@artifactId@-@version@",
+  "nodeId": "localhost",
+  "descriptor": {
+    "exec": "java -Dport=%p -jar ../mod-spring-template/target/mod-spring-petstore.jar -Dhttp.port=%p embed_postgres=true"
```

Comment:
Is it possible to use the replacement tokens here too.

Reply:
Could you please explain a bit about that? What should be replaced?

Reply:
"mod-spring-petstore" with "@artifactId@"

Also "mod-spring-template" is incorrect, and perhaps could use that Maven replacement token too.

Reply:
IIUC then "embed_postgres" is not used anymore.

Reply:
Done